.github/workflows/wled-ci.yml

```diff
@@ -39,6 +39,11 @@ jobs:
         environment: ${{ fromJSON(needs.get_default_envs.outputs.environments) }}
     steps:
     - uses: actions/checkout@v3
+    - name: Set up Node.js
+      uses: actions/setup-node@v4
+      with:
+        cache: 'npm'
+    - run: npm install
```

Comment:
Maybe add "npm audit fix" as another step?

It tries to update components with know security vulnerabilities, unless there are "breaking changes".

Reply:
I would rather manually execute `npm audit fix` and then commit the new `package-lock.json`, so that everyone has the fixes for security vulnerabilities.
I was planning to rework `cdata.js` more thoroughly in the future anyway and update all dependencies. This would also give me the opportunity to run an `npm audit fix`. But I would have done that in a separate PR.